satellite/satellitedb/accounting.go

```
@@ -38,20 +39,20 @@ func (db *accountingDB) SaveGranulars(ctx context.Context, logger *zap.Logger, l
 	// Any change in these assumptions would result in a change to this function
 	// in particular, we should consider finding the sum of bwagreements using SQL sum() direct against the bwa table
 	if len(bwTotals) == 0 {
-		logger.Warn("In SaveGranulars with empty bwtotals")
+		logger.Warn("In SaveBWRaw with empty bwtotals")
 		return nil
 	}
 	//insert all records in a transaction so if we fail, we don't have partial info stored
 	tx, err := db.db.Open(ctx)
 	if err != nil {
-		logger.DPanic("Failed to create DB txn in tally query")
+		logger.DPanic("Failed to create DB txn in SaveBWRaw")
 		return err
 	}
 	defer func() {
 		if err == nil {
 			err = tx.Commit()
 		} else {
-			logger.Warn("DB txn was rolled back in tally query")
+			logger.Warn("DB txn was rolled back in SaveBWRaw")
 			err = tx.Rollback()
```

Do not hide the actual error.